Documentation/install/azure/azure-private.md

```diff
@@ -0,0 +1,62 @@
+# Azure private clusters
+
+Use Tectonic Installer to create clusters within an existing private Azure network. In addition to the standard Azure requirements, private clusters also require:
+* An existing Virtual Network with a pre-configured and connected VPN link.
+* A DNS domain pre-registered into Azure DNS, or a 3rd party DNS server that supports DNS UPDATE operations.
+
+Once these requirements are met, follow the guide to [Install Tectonic on Azure with Terraform][azure-terraform].
+
+The machine running Tectonic Installer must be behind the VPN connection to the VNet. This machine must have direct IP connectivity to the VMs in the cluster VNet.
+
+## Connected VPN link
+
+Private Azure networks require that an external VPN be available and running prior to running Tectonic Installer.
+
+For best results, use an OpenVPN connection to an OpenVPN Access Server that is deployed into the cluster VNet.
+
+For more information, see the Microsoft Azure document [Deploy an Openvpn Access Server][deploy-openvpn].
+
+## DNS domain or server
+
+Private clusters must use an external DNS provider, rather than Azure’s default hostname resolution.
```

Comment:
Line 5 above says to configure a domain in Azure DNS -- this contradicts. Can you clarify?

Reply:
my understanding of 'a DNS domain pre-registered into Azure DNS' was not the same as 'configured in Azure DNS'. I'm turning to @alexsomesan for clarification.